pkg/report/sarif.go

```diff
@@ -346,8 +346,18 @@ func ToPathUri(input string, resultClass types.ResultClass) string {
 	return clearURI(input)
 }
 
+// clearURI clears URI for misconfigs
 func clearURI(s string) string {
-	return strings.ReplaceAll(strings.ReplaceAll(s, "\\", "/"), "git::https:/", "")
+	s = strings.ReplaceAll(s, "\\", "/")
+	if strings.HasPrefix(s, "git::https:/") {
+		s = strings.ReplaceAll(s, "git::https:/", "")
```

Comment:
`.git` needs to be handled here too. The URI is as follows: `git::https://example.com/vpc.git?ref=v1.2.0`

Reply:
hm.. nice catch!
I didn't check previous logic.
Updated in https://github.com/aquasecurity/trivy/pull/7898/commits/3ebdecfdafc7d5baf56816c30338cbf23db59a82